src/sparse/KokkosSparse_sptrsv_handle.hpp

```diff
@@ -43,17 +43,29 @@
 
 #include <Kokkos_MemoryTraits.hpp>
 #include <Kokkos_Core.hpp>
+
 #include <iostream>
 #include <string>
 
 #ifndef _SPTRSVHANDLE_HPP
 #define _SPTRSVHANDLE_HPP
 
+#if defined(KOKKOSKERNELS_ENABLE_TPL_CBLAS)   && \
+    defined(KOKKOSKERNELS_ENABLE_TPL_LAPACKE) && \
+   (defined(KOKKOSKERNELS_ENABLE_TPL_SUPERLU) || \
+    defined(KOKKOSKERNELS_ENABLE_TPL_CHOLMOD))
+
+ // Enable supernodal sptrsv
+ #define KOKKOSKERNELS_ENABLE_SUPERNODAL_SPTRSV
+ #include <KokkosSparse_CrsMatrix.hpp>
+
+#endif
+
 namespace KokkosSparse {
 namespace Experimental {
 
 // TP2 algorithm has issues with some offset-ordinal combo to be addressed
```

Comment:
What does this comment mean?  If some combinations of types don't work, you can use `static_assert` to make them fail at compile time.